src/data/constants.js

```diff
@@ -3,6 +3,7 @@ export const LOGIN_PAGE = '/login';
 export const REGISTER_PAGE = '/register';
 export const RESET_PAGE = '/reset';
 export const DEFAULT_REDIRECT_URL = '/dashboard';
+export const ENTERPRISE_LOGIN_URL = '/enterprise/login';
```

Comment:
This is an edX specific endpoint, we should not hard code it here and instead pull it from remote config.